pip/commands/install.py

```diff
@@ -266,6 +276,7 @@ def run(self, options, args):
                     "continue."
                 )
             install_options.append('--home=' + temp_target_dir)
+            install_options.append('--prefix=')
```

Comment:
I do not fully understand, how `install_options` and `distutils_scheme()` in [locations.py](https://github.com/pypa/pip/pull/4111/files#diff-f21e4e94baf6a2d2f98faab0451e418a) do work together, but it seems to work this way.